bunsen-spark/src/main/java/com/cerner/bunsen/spark/Bundles.java

```diff
@@ -219,6 +220,47 @@ public static Bundles forR4() {
     return spark.createDataFrame(resourceRdd.rdd(), converter.getSchema());
   }
 
+  /**
+   * Extracts the given resource type from the RDD of bundles and returns
+   * it as a Dataset of that type, including any declared resources contained
+   * to the parent resource.
+   *
+   * @param spark the spark session
+   * @param bundles the RDD of FHIR Bundles
+   * @param resourceClass the type of resource to extract.
+   * @param containedClasses the type of the resources contained to the
+   *      parent resource.
+   * @return a dataset of the given resource
+   */
+  public Dataset<Row> extractEntry(SparkSession spark,
+      JavaRDD<BundleContainer> bundles,
+      Class resourceClass,
+      Class... containedClasses) {
```

Comment:
I think we'll need to make this parameter an explicit list. In practice, Java will have difficulty dispatching between this new method and [`extractEntry`](https://github.com/cerner/bunsen/blob/0ff8cb8973c743f1b8de98c322061271b489d721/bunsen-spark/src/main/java/com/cerner/bunsen/spark/Bundles.java#L186) that has the single Class parameter since it can't tell if the call to `extractEntry` without a final argument is a call without contained classes, or just with empty contained classes.

Reply:
Fixed f0d5fdcc5c60ac12ae32e0b907dfdea91ab08fd8